lib/kuzzle/kuzzle.ts

```diff
@@ -823,9 +823,8 @@ class Kuzzle extends KuzzleEventEmitter {
         inString = stringify(input);
     }
 
-    return murmur(
+    return murmur.v3(
```

Comment:
This library generates 32-bit hashes instead of the 128-bit ones we had previously.
See https://www.npmjs.com/package/murmurhash

Reply:
Okay, you are right i found one that uses 128Bit w/o binaries. But i wonder if this is really usefull ?

Reply:
It is not really indeed.